repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -135,8 +135,16 @@ class Interpreter(shellPrompt0: Ref[String],
     def search(target: scala.reflect.runtime.universe.Type) = Interpreter.this.compiler.search(target)
     def compiler = Interpreter.this.compiler.compiler
     def newCompiler() = init()
-    def history = history0.toVector.dropRight(1)
+    def history = history0
     def show[T](a: T, lines: Int = 0) = ammonite.pprint.Show(a, lines)
+
+    implicit val historyPPrinter: pprint.PPrinter[History] = 
```

Comment:
The is not get picked up in the companion (it should be according to [stackoverflow](http://stackoverflow.com/questions/5598085/where-does-scala-look-for-implicits)). Where should I put it? It is a bit hard to access from `UnitTests` if it's in `ReplApi`.